consent-ws/src/main/resources/assets/api-docs.yaml

```diff
@@ -522,6 +522,8 @@ paths:
           description: Returns the requested consent.
           schema:
             $ref: '#/definitions/Consent'
+        '400':
+          description: The election associated to the given consent is not archived.
```

Comment:
Please add the new consent fields to `'#/definitions/Consent'`

Also, I think the language here could be more clear. The way I read this, the conditions for `lastElectionArchived` are that:
1. an election exists for this consent
2. that election has not been archived
3. therefore that election is in some kind of state that connotes open-ness. 

If that's correct, then I think the language could be more like
```
Consent updates are not allowed with an open election.
```

Reply:
The election can be Canceled and NOT archived, or Closed and NOT archived, in those cases the consent cannot be updated either.

Reply:
OK, those cases are very confusing to me. As a user in ORSP (the only client that would encounter this at the moment), it doesn't make much sense to me what a closed+unarchived election or canceled+unarchived election really means. I think we need better language around all of the possible error cases.

Reply:
If the last election related to that consent is not archived or Open, the consent cannot be updated.

Reply:
Thanks Walter! Then my suggestion for language would be something along the lines of:
```
Consent updates are only allowed when:
1. there have been no previous elections created for this consent
2. the most recent election has been closed/cancelled AND archived
```

Reply:
`'#/definitions/Consent'` is now reflecting both new consent fields.
Also refactored the error description as mentioned above.